protobufs/entity/service.proto

```diff
@@ -48,6 +48,15 @@ service DocumentService {
       description: "Get the current entity"
     };
   }
+  // Entity Relation Get
+  rpc GetEntityByRelationship(GetRequestRelationship) returns (EntityResponse) {
+    option (google.api.http) = {
+      get: "/entity/{identifier}/relationship/{relationship_identifier}"
```

Comment:
First I wanted to pass it as normal GET parameters. 
`/relationship?relationship_identitfier=val1`

What do we prefer?

Reply:
couldn't we do `get/relationship/{relationship_id}/entity` ?

Reply:
mhm. We would break the pattern a bit. 

@lucasvo do you have a preference?

Reply:
This is what we discussed in a meeting. We agreed on:
`/entity/{identifier}/relationship/{relationship_identifier}`

Reply:
Actually, just talking to @charleenfei i misunderstood the http method. The call should be:
`relationship/{relationship_identifier}/entity`